drivers/amazon/create.go

```diff
@@ -139,6 +139,12 @@ func (p *provider) create(ctx context.Context, opts autoscaler.InstanceCreateOpt
 		},
 	}
 
+	if p.imdsTokens != "" {
```

Comment:
Nit: use the same convention as others instead of directly modifying in.

```
var in *ec2.InstanceMetadataOptionsRequest

if p.imdsTokens != "" {
    metadataOptions = &ec2.InstanceMetadataOptionsRequest{
          HttpTokens: aws.String(p.imdsTokens),
    }
}

```